### DISCUSSION TITLE
Highlighting lifecycle/development stage more prominently

### DISCUSSION THREAD

**Discussion by TimTaylor** [epiverse-trace/epiverse-trace.github.io#80] [General]
@Bisaloo and @adamkucharski gave a nice talk at UKHSA yesterday which may encourage a few people to click on the [GitHub landing page](https://github.com/epiverse-trace). One thing that jumped out to me is that there is nothing that communicates package lifecycles on the landing page or in the "About" descriptions of the individual packages. Also when clicking through to the individual packages the badges do not really jump out at you.  I wonder if there sure be a more prominent way to convey this information, particularly for packages going through very early development . Likewise - should more mature packages (e.g. finalsize) be highlighted as such?

**Comment by adamkucharski** [comment]
This is a useful point. Think we could also do more with the landing page README to list information in order of likely user need states (e.g. wanting to know what we do, make use of more mature packages, know how to contribute etc.)

**Comment by Bisaloo** [comment]
This is on my radar and I plan to address it as part of the "landing page for packages" (Task 16 in https://github.com/orgs/epiverse-trace/projects/24).

It might take a little bit of time to get to it since it's competing with other priorities so any short-term solution in the meantime may be useful.

**Reply by TimTaylor** [reply]
A short-term and low-effort quick win would be for maintainers to add something to the "About" section that conveys the additional information that then shows up on the landing page:  E.g. something like epidemics may have something akin to:

[**_Not published - under active development_**]. A library of compartmental epidemic models that include demographic structure, non-pharmaceutical interventions, and vaccination regimes.

**Reply by pratikunterwegs** [reply]
Happy to implement this suggestion as well. Might be a bit silly, but we could also standardise use of emojis to represent lifecycle stages until we work out a better solution.

**Reply by Bisaloo** [reply]
Emojis could be nice as a way to quickly identify the lifecycle but one big hurdle is that their interpretation is far from universal and there is a strong cultural factor which may create unexpected confusion.

**Reply by pratikunterwegs** [reply]
Would a traffic light system be widely understood? Essentially some way of transferring the colours from the status badges to the About section?

**Reply by TimTaylor** [reply]
Think words are better:
(a) flexibility to communicate more;
(b) users do not have to read something else to understand meaning;
(c) accessibility - colour blindness (unless you recreated entire traffic light and these differ globally :sweat_smile: )

**Comment by pratikunterwegs** [comment]
Thanks @TimTaylor for noticing this and the suggestions, as @Bisaloo says there's a task already open and I'm happy to help with implementing improvements to the landing page as well. The individual pages do have badges that it should be possible to transfer - alternatively, having a dedicated ET website is another solution - happy to help with that as well.

**Comment by Bisaloo** [comment]
Related: https://github.com/epiverse-trace/epiverse-trace.r-universe.dev/issues/8

**Comment by avallecam** [comment]
Does the topic on [substantiable dashboards for open-source software projects](https://github.com/SciKit-Surgery/scikit-surgery-stats#scikit-surgery-stats) related to this question? more details in [this abstract](https://rslondon.ac.uk/rslondonsoutheast2023/abstracts/#ft6).
